src/main/webapp/app/shared/layouts/navbar/navbar.component.ts

```diff
@@ -126,4 +158,11 @@ export class NavbarComponent implements OnInit, OnDestroy {
             }
         }
     }
+
+    /**
+     * check if exam already started
+     */
+    examHasStarted(exam: Exam): boolean {
```

Comment:
you could also check if it is already over so that its only between start and due date.

Reply:
makes, total sense, thanks 👍